tools/alphafold/alphafold.xml

```diff
@@ -76,7 +77,7 @@
                 <param name="fasta_file" type="data" format="fasta" label="Fasta file from history" help="Select single fasta protein sequence from your history. If you wish to fold multiple proteins, submit an individual job for each protein." />
             </when>
             <when value="textbox">
-                <param name="fasta_text" type="text" area="true" value="" label="Fasta sequence" help="Paste single protein sequence in the textbox above. DO NOT include a fasta header line (starting with '>'). If you wish to fold multiple proteins, submit an individual job for each protein.">
+                <param name="fasta_text" type="text" area="true" value="" label="Sequence" help="Paste single protein sequence into the textbox. DO NOT include a fasta header line (starting with '>'). If you wish to fold multiple proteins, submit an individual job for each protein.">

```

Comment:
If no faster header is allowed, why are so many different characters `valid`?

Reply:
You can also use `area="true"` https://docs.galaxyproject.org/en/latest/dev/schema.html#id37

But not sure if useful :)

Reply:
Great point. I will make a validator for IUPAC characters :)